parm/config.wcoss2.yaml

```diff
@@ -27,15 +27,11 @@ marinedump:
   providers:
     ghrsst:
       list:
-        - sst_viirs_n21_l3u
         - sst_viirs_n20_l3u
+        - sst_viirs_n21_l3u
         - sst_viirs_npp_l3u
-        - sst_avhrrf_ma_l3u
         - sst_avhrrf_mb_l3u
         - sst_avhrrf_mc_l3u
```

Comment:
since we're going to create a 2020-present database, check that we list all the satellites. We're missing metop-a for example.